main.py

```diff
@@ -1,44 +1,46 @@
 import time
 from selenium import webdriver
-from webdriver_manager.chrome import ChromeDriverManager 
+from webdriver_manager.chrome import ChromeDriverManager
 from webdriver_manager.firefox import GeckoDriverManager
-
+from colored import fg, bg, attr  # pip install colored
 # Initiliaze Webdriver
 try:
-	driver = webdriver.Chrome(ChromeDriverManager().install())
+    driver = webdriver.Chrome(ChromeDriverManager().install())
 except:
-	driver = webdriver.Firefox(executable_path=GeckoDriverManager().install())
+    driver = webdriver.Firefox(executable_path=GeckoDriverManager().install())
 
-def FacebookLogin():
-    #Enter Your Email ID And Password
-	user=input('Enter Email Id:')  
-	password=input('Enter Password:')
-
-	#Opening Facebook.
-	driver.get('https://www.facebook.com/') 
-	print ("Facebook Opened") 
-	time.sleep(1) 
-	  
-    #Entering Email and Password
-	username_box = driver.find_element_by_id('email') 
-	username_box.send_keys(user) 
-	print ("Email Id entered") 
-	time.sleep(1) 
-	  
-	password_box = driver.find_element_by_id('pass') 
-	password_box.send_keys(password) 
-	print ("Password entered") 
-
-	#Pressing The Login Button  
-	login_box = driver.find_element_by_id('loginbutton') 
-	login_box.click() 
-
-	  
-	print ("Done") 
-	input('Press anything to quit') 
-	driver.quit() 
-	print("Finished")
 
-FacebookLogin()
+def FacebookLogin():
+    try:
+        # Enter Your Email ID And Password
+        user = input(f"{fg('green_1')}Enter Email Id:{attr('reset')}")
```

Comment:
You can face an error called TABERROR. This is caused when you use spaces and tabs inconsistently. To solve it install Untabify from VS Code marketplace and use its keyboard shortcut to fix the error